webapp/channels/src/actions/telemetry_actions.jsx

```diff
@@ -20,31 +18,6 @@ export function shouldTrackPerformance(state = store.getState()) {
     return isDevModeEnabled(state) || isTelemetryEnabled(state);
 }
 
-export function trackEvent(category, event, props = {}) {
-    const state = store.getState();
-    if (
-        isPerformanceDebuggingEnabled(state) &&
-        getBool(state, Preferences.CATEGORY_PERFORMANCE_DEBUGGING, Preferences.NAME_DISABLE_TELEMETRY)
```

Comment:
Also, we can remove the `Disable telemetry events sent from the client` user setting as well